SimpleCrypto/Implementation/ICryptography/SimpleCryptoImplementation.cpp

```diff
@@ -0,0 +1,318 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2023 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../Module.h"
+
+#include <interfaces/ISimpleCrypto.h>
+#include <interfaces/IConfiguration.h>
+#include <interfaces/ICryptography.h>
+
+#ifdef USE_OPENSSL
```

Comment:
This, is, or should be :-), part of the cryptalgo/cryptalgo.h you include here. If you enable the OpenSSL during the build of the Cryptalgo unit the random generator is (or should be) taken from openssl... So I am hping we do not need any openssl awareness in here?